.chloggen/enrich_azuremonitor_tags.yaml

```
@@ -4,13 +4,13 @@
 change_type: enhancement
 
 # The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
-component: receiver/hostmetrics
+component: azuremonitorexporter
```

To fix the changelog job failure
```suggestion
component: exporter/azuremonitor
```